lib/layout.js

```diff
@@ -174,11 +172,21 @@ const pickZone = (opts, frameBounds, targetBounds, size) => {
     { side: `end`, standing: `below`, flow: `column`, order: 1, w: f.x2, h: (f.y2 - t.y2) },
     { side: `start`, standing: `left`, flow: `row`, order: -1, w: t.x, h: f.y2 },
   ]
-
+  
+  /* Order the zones by the amount of popup that would be cut out if that zone is used.
+     The first one in the array is the one that cuts the least amount.
+      
+     const area = size.w * size.h  // Popup area is constant and it does not change the order
+  */
+  zones.forEach((z) => {   
```

Comment:
Thanks, makes sense as a lightweight improvement. To add to this discussion, I don't think I would consider this a solution to #14 which I think deserves more robust dedicated consideration. I know that was not your intent here but I think its worth referencing that old issue. I don't know what a more robust algorithm would be exactly but I can imagine that for instance it would know to treat a absolutely smaller area that is more balanced on both dimensions as preferable. This warrants a diagram to make clear but hopefully the hint has been successfully suggested. Anyways, again, thanks.